### DISCUSSION TITLE
Release 2.0.0a2 - alpha quality, for testing only

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbackup/borg#6799] [General]
borgbackup 2.0.0a2 alpha release - do not use this on production backup repositories.

Long changelog:
https://borgbackup.readthedocs.io/en/2.0.0a2/changes.html#version-2-0-0a2-2022-06-26

If you use pip to install this, use: pip install "borgbackup==2.0.0a2"

Docs: https://borgbackup.readthedocs.io/en/2.0.0a2/

Feedback: here or in the issue tracker.

<hr /><em>This discussion was created from the release <a href='https://github.com/borgbackup/borg/releases/tag/2.0.0a2'>Release 2.0.0a2 - alpha quality, for testing only</a>.</em>

**Comment by ThomasWaldmann** [comment]
When trying `borg transfer`, this might be useful:

https://raw.githubusercontent.com/borgbackup/borg/borg2/docs/usage/transfer.rst

**Comment by real-yfprojects** [comment]
How much change to the command line interface can be expected until the stable release?

**Comment by ThomasWaldmann** [comment]
well considering we are still in an early phase of the release/testing process, it is a bit hard to tell, but i guess the most fundamental known stuff is already done:

• no scp style repos, use ssh://…
• separation of repo and archive name, no ::
• separation of commands working on repos and on archives
• more commands will offer -a
• less commands offer positional archive names if it can be also done with -a

**Comment by ThomasWaldmann** [comment]
PSA: I just noticed that I had placed a rather hard restriction on the repository version this borg wants to accept (this was mainly to avoid accidental damage, so this alpha quality borg is not run against production repos):

It only accepts `version = 2` repos (see the `repo_dir/config` file). It also uses version 2 for new repos.

But, this currently also applies to `--other-repo` (which is used to give the **source** repository for `borg transfer`).

Thus, if you want to test `borg transfer` with borg 2.0.0a2, it means that you first have to edit the repo config of the repository made with borg 1.2.
